website/blog/2022-01-13-docusaurus-2021-recap/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Docusaurus 2021 Recap
+authors: [Josh-Cena]
+tags: [recap]
+image: ./img/thumbnail.png
+---
+
+Docusaurus has made great progress in 2021! We've **reached full feature parity** with v1 after [the release of internationalization](../2021-03-09-releasing-docusaurus-i18n.md), [announced our **first beta**](../2021-05-12-announcing-docusaurus-two-beta/index.md), and welcomed **significantly more users** this year.
+
+<!--truncate-->
+
+## Highlights
+
+Let's begin by going over a few highlights this year. Remember the [todo-list](../2021-05-12-announcing-docusaurus-two-beta/index.md#whats-next) from half a year ago? Time to take it out again and see how far we've come!
+
+- **Internationalization**: officially released four months after its initial landing in late 2020. Seen great success in [many projects](/showcase?tags=i18n)
+- **Autogenerated sidebars**: no need to maintain a big `sidebars.js` anymore
+- **Webpack 5**: significantly improved build performance
+- **Trailing slash config**: seamless integration with any content hosting provider
+- **Redesigned mobile navigation**: mobile users now enjoy all navigation features as on desktop
+- **UI improvements**: more accessible quotes, better admonitions, back-to-top button...
+- **Blog multi-authors**: credit all the amazing contributors that collaborated on one blog post!
+- **More blog features**: archive page, structured data for SEO, full post content in feeds – empowering you to ship a fully-featured blog in no time
```

Comment:
@yangshun FYI, although I don't have strong opinions on this, em-dashes are usually used without spaces around them. The AP style guide is the only guide I know that recommends spaces around them. I've just changed to using en-dashes instead because " – " === "—", if you think spaces improve legibility.

Reply:
Ah I see. We've been putting spaces around our dashes (for e.g. a few instances on this page https://docusaurus.io/docs). I think legibility is most important, so as long as there are spaces it's good 😄

Reply:
Yes, I think legibility is a legitimate point :D